test/helpers/kubectl.go

```diff
@@ -0,0 +1,463 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/asaskevich/govalidator"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/cilium/cilium/api/v1/models"
+	"github.com/cilium/cilium/pkg/k8s"
```

Comment:
Remove this dependency. This dependency has a transitive dependency of pkg/bpf which makes it impossible to run Ginkgo on a Mac setup, which is required for these tests. The only use of this pkg is lines 132-133. I'm OK with duplicating those constants for now and then adding a TODO / FIXME and explain why they are not being pulled in from the pkg/k8s pkg.